.gitignore

```diff
@@ -0,0 +1,9 @@
+# Ignore files generated by common IDEs
+.idea
+.vscode
+
+# Ignore OS generated files
+.DS_Store*
+ehthumbs.db
+Thumbs.db
+._*
```

Comment:
Should we include `node_modules`?

```suggestion
._*

# Ignore Node generated files.
node_modules

```

Reply:
This directory shouldn't have node_modules in it. It is a base theme and all node packages would be installed the in generated child theme outside of this code. I think we are okay to leave it out unless there becomes a use case.